src/modules/menu/components/menu/feedback/FeedbackButton.vue

```diff
@@ -19,21 +19,39 @@
         v-if="showFeedbackForm"
         :title="$t('test_map_give_feedback')"
         fluid
-        @close="showFeedbackForm = false"
+        @close="closeAndCleanForm"
     >
         <div class="p-2" data-cy="feedback-form">
             <span>{{ $t('feedback_rating_title') }}</span>
             <FeedbackRating
                 class="my-4 text-center"
                 :max-rating="maxRating"
+                :disabled="request.completed"
```

Comment:
Good point, in the end I also cleared the form when a successful request had been sent, so that only the "Your message was sent successfully..." text is present.
I've therefore only disabled inputs on `request.pending`